lib/v2/screens/transfer/send/send_enter_data/interactor/mappers/send_transaction_mapper.dart

```diff
@@ -20,6 +22,8 @@ class SendTransactionMapper extends StateMapper {
       var selectedFiat = settingsStorage.selectedFiatCurrency;
       String fiatAmount = currentState.ratesState.fromSeedsToFiat(parsedQuantity, selectedFiat).fiatFormatted;
 
+      eventBus.fire(TransactionSentEventBusEvent(resultResponse.transactionModel));
```

Comment:
Should the eventBus be inside the if else? if all-Results is not success we may send null data

Reply:
No we are checking for error above